art/dashboard/app.py

```diff
@@ -22,8 +22,8 @@
 if not LOGS_PATH.exists():
     raise ValueError(f"Path {LOGS_PATH} does not exist.")
 app = Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
-ORDERED_STEPS = prepare_steps(LOGS_PATH)
 STEPS_INFO = prepare_steps_info(LOGS_PATH)
+ORDERED_STEPS = [x for x in prepare_steps() if x in STEPS_INFO.keys()]
```

Comment:
If these changes are applied I would change the name STEPS_INFO to existing steps or something, that differentiate ORDERED_STEPS from STEPS_INFO - first are actually the hardcoded ones, and the other are the existing ones. For now the names suggest we're talking about the same steps, but with and without order